### ISSUE TITLE
ImportError: Unable to import required dependencies: numpy: 

### ISSUE THREAD

**Issue by boazdori** [numpy/numpy#15090]
when running the bellow code or any code having import pandas as pd I get the above error.
this is happening in the vs code ide run cod execution
I can run the code as run python file in terminal
for some reason to the output using run code I am getting the above error.
### Reproducing code example:

```python
import pandas as pd

x = list(range(0,100))
pd.DataFrame(x)

print(x)
```
### Error message:
Traceback (most recent call last):
  File "c:\Users\user\Documents\Visual Studio 2017\Projects\test1.py", line 7, in <module>
    import pandas as pd
  File "C:\Users\user\Anaconda3\lib\site-packages\pandas\__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
numpy: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy c-extensions failed.
- Try uninstalling and reinstalling numpy.
- If you have already done that, then:
  1. Check that you expected to use Python3.7 from "C:\Users\user\Anaconda3\python.exe",
     and that you have no directories in your PATH or PYTHONPATH that can
     interfere with the Python and numpy version "1.17.4" you're trying to use.
  2. If (1) looks fine, you can open a new issue at
     https://github.com/numpy/numpy/issues.  Please include details on:
     - how you installed Python
     - how you installed numpy
     - your operating system
     - whether or not you have multiple versions of Python installed
     - if you built from source, your compiler versions and ideally a build log

- If you're working with a numpy git repository, try `git clean -xdf`
  (removes all files not under version control) and rebuild numpy.

Note: this error has many possible causes, so please don't comment on
an existing issue about this - open a new one instead.

Original error was: DLL load failed: The specified module could not be found.

### Numpy/Python version information:
numpy=1.17.4
python 3.7.4

I am using anaconda
I tried to remove and reinstall numpy

**Comment by rgommers** [comment]
Maybe you're not in the conda environment you're expecting? What does `conda list` say? If it has `numpy` and `python` in it, does simply `python -c "import numpy"` on the command line work?

**Comment by boazdori** [comment]
numpy and python in the conda list
running python -c "import numpy"  works fine 
the problem occurs only when running the code in vs code using the run cod method -- upper right-hand run button.
running the code in vs code using run python file to terminal executed fine

**Comment by rgommers** [comment]
That means your problem is that VS Code is using a different Python than you are getting when you execute `python` on the command line. I suggest looking into the VS Code docs for that.

**Comment by JulianSMoore** [comment]
Info from someone similarly affected: I had an Anaconda environment but had to install Tensorflow 2 by PIP (not available from Conda); mixing PIP and Conda led to duplicate packages including numpy 1.17.3 and 1.17.4 - I built a fresh environment with minimal Conda and the rest by PIP (including TF2 GPU) and after rebuilding the VS studio project the issue was resolved - numpy now imports OK. Don't know if there was a better way, but asking whether there are duplicate packages (and/or whether TF2 has been installed - it brings in a lot of dependencies) would seem like a good start.  HTH

**Comment by rgommers** [comment]
@JulianSMoore that's a different issue you were having. And conda defaults has had Tensorflow 2.0 for over a month now. But yes you're right that defaults can be behind. If `numpy` is already installed in a conda env though, `pip` should not install a second copy.

**Comment by rgommers** [comment]
I'll close this, since the original question was answered and there's no further follow-up.

**Comment by GeorgeMKhoury** [comment]
For anyone else who runs into this issue and finds this thread: Not an issue with numpy, but rather with VS code. You need to tell VS Code to activate the appropriate conda environment in settings: Python>Terminal:Activate Environment. Note that if you install anaconda with default settings, this does not work properly if your terminal is git bash, but it works fine if you use cmd (and the integrated terminal).

**Comment by BarbacilNico** [comment]
Hi, I solved this problem by updating numpy to 1.18.3 from anaconda console

**Comment by cbbuaa** [comment]
I solved this problem by selecting the default shell as command prompt.
![1](https://user-images.githubusercontent.com/23333414/85951861-7823c880-b998-11ea-8798-5c064c566541.png)
